lib/ffeconway.gi

```diff
@@ -1184,33 +1184,6 @@ InstallMethod( LogFFE,
         [ IsFFE and IsCoeffsModConwayPolRep, IsFFE and IsInternalRep],
         DoDLog );
 
-#############################################################################
-##
-#M  Order -- multiplicative order of an FFE
-##
-
-
-InstallMethod( Order,
```

Comment:
Where is the code that is making this one redundant?


Also, can you please apply the same change to `hpcgap/lib/ffeconway.gi`?

Reply:
The method is duplicated in `ffe.gi:779`. And yes, will do!